api/v1alpha3/openstackmachine_types.go

```diff
@@ -57,6 +57,10 @@ type OpenStackMachineSpec struct {
 	// A networks object. Required parameter when there are multiple networks defined for the tenant.
 	// When you do not specify the networks parameter, the server attaches to the only network created for the current tenant.
 	Networks []NetworkParam `json:"networks,omitempty"`
+
+	// IP address of a port from this subnet will be marked as AccessIPv4 on the created compute instance
+	AccessSubnetUUID string `json:"accessSubnetUuid,omitempty"`
```

Comment:
Other variables does not start with Access. `Subnet` is easy to understand.

Reply:
renamed AccessSubnetUUID to Subnet

Reply:
@jsen- Thanks updating.